src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -148,7 +148,7 @@ CrashInfo::GatherCrashInfo(MINIDUMP_TYPE minidumpType)
         return false;
     }
     // Gather all the module memory mappings (from /dev/$pid/maps)
-    if (!EnumerateModuleMappings())
+    if (!EnumerateMemoryRegions())
```

Comment:
I meant to ask what was the difference between the calls. Now on apple we are calling EnumerateMemoryRegions twice (140 and this line)

Reply:
No, it is only called on line 140. Line 151 is in the #else of the #ifdef __APPLE__